### PR TITLE
RHSTOR-1643: Fix Import Order

### PR DESCRIPTION
**Pull request by rexagod**
Introduces rules that define the order of imports.

Refer: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md

Fixes: https://issues.redhat.com/browse/RHSTOR-1643

### CODE REVIEW

**Comment by rexagod** [comment]
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_console/8131/pull-ci-openshift-console-master-ceph-storage-plugin/1359775144951681024#1:build-log.txt%3A560

With respect to the artifacts found [here](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_console/8131/pull-ci-openshift-console-master-ceph-storage-plugin/1359775144951681024/artifacts/ceph-storage-plugin/test/artifacts/gui_test_screenshots/cypress/videos/), the failed build seems to be because of some flakes mentioned below:

`cy.login`: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_console/8131/pull-ci-openshift-console-master-ceph-storage-plugin/1359775144951681024/artifacts/ceph-storage-plugin/test/artifacts/gui_test_screenshots/cypress/videos/ocs-presistent-dashboard.spec.ts.mp4

`system-name-mcg` selector not found in DOM: https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_console/8131/pull-ci-openshift-console-master-ceph-storage-plugin/1359775144951681024/artifacts/ceph-storage-plugin/test/artifacts/gui_test_screenshots/cypress/videos/noobaa-sso.spec.ts.mp4

File: frontend/packages/ceph-storage-plugin/.eslintrc [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "extends": ["../.eslintrc.js"],
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/.eslintrc:2]: Extend on the `eslint-plugin-console`

**Review by bipuladh — CHANGES_REQUESTED** [review]

File: frontend/packages/ceph-storage-plugin/.eslintrc (hunk 1 above)
Comment (rexagod) [reply frontend/packages/ceph-storage-plugin/.eslintrc]: Fixed at https://github.com/openshift/console/pull/8131/commits/142af325d361bfaba9878faa6abcbbadee0085f5#diff-e77a554f3d40cd16fe423926aef6d9fc75d90579de71b65f621a5302794396ddR3. PTAL.

**Review by bipuladh — APPROVED** [review]

**Pull request merged**